src/parser.sid

```diff
@@ -205,6 +263,32 @@
 		<err-expected-type>;
 	};
 
+        directive-properties: (dialect: re-dialect) -> () [
```

Comment:
That's `(dialect :re-dialect)` - the `:` is attached to the type.

Reply:
Hmm... that's confusing.  it switches from the sid file to the act file